server/controllers/activity.js

```diff
@@ -32,7 +32,7 @@ const getActivities = (req, resp) => {
   const {
     user: { _id: userId }
   } = req;
-  const sanitizedPage = sanitize(page);
+  const sanitizedPage = +sanitize(page);
```

Comment:
what is this plus sign?

Reply:
sanitizing convert number to string so this is quick way to convert it back to number

Reply:
const sanitizedPage = Number(page);